libopenage/error/gl_debug.cpp

```diff
@@ -10,7 +10,7 @@ namespace openage {
 namespace error {
 
 namespace {
-void callback(GLenum source, GLenum, GLuint, GLenum, GLsizei, const GLchar *message, const void*) {
+void APIENTRY callback(GLenum source, GLenum, GLuint, GLenum, GLsizei, const GLchar *message, const void*) {
```

Comment:
It must be (I'm assuming) in `epoxy/gl.h`. I was relying on the CI reports to ensure that it is.